templates/parts/isu-navbar.html.twig

```diff
@@ -15,51 +15,51 @@
+        <li><a href="https://www.iastate.edu/index/D/">D</a></li>
+        <li><a href="https://www.iastate.edu/index/E/">E</a></li>
+        <li><a href="https://www.iastate.edu/index/F/">F</a></li>
+        <li><a href="https://www.iastate.edu/index/G/">G</a></li>
+        <li><a href="https://www.iastate.edu/index/H/">H</a></li>
+        <li><a href="https://www.iastate.edu/index/I/">I</a></li>
+        <li><a href="https://www.iastate.edu/index/J/">J</a></li>
+        <li><a href="https://www.iastate.edu/index/K/">K</a></li>
+        <li><a href="https://www.iastate.edu/index/L/">L</a></li>
+        <li><a href="https://www.iastate.edu/index/M/">M</a></li>
+        <li><a href="https://www.iastate.edu/index/N/">N</a></li>
+        <li><a href="https://www.iastate.edu/index/O/">O</a></li>
+        <li><a href="https://www.iastate.edu/index/P/">P</a></li>
+        <li><a href="https://www.iastate.edu/index/Q/">Q</a></li>
+        <li><a href="https://www.iastate.edu/index/R/">R</a></li>
+        <li><a href="https://www.iastate.edu/index/S/">S</a></li>
+        <li><a href="https://www.iastate.edu/index/T/">T</a></li>
+        <li><a href="https://www.iastate.edu/index/U/">U</a></li>
+        <li><a href="https://www.iastate.edu/index/V/">V</a></li>
+        <li><a href="https://www.iastate.edu/index/W/">W</a></li>
+        <li><a href="https://www.iastate.edu/index/X/">X</a></li>
+        <li><a href="https://www.iastate.edu/index/Y/">Y</a></li>
+        <li><a href="https://www.iastate.edu/index/Z/">Z</a></li>
       </ul>
     </li>
     <li class="isu-navbar_break" aria-hidden="true"></li>
-    <li><a href="http://info.iastate.edu/">Directory</a></li>
-    <li><a href="http://www.fpm.iastate.edu/maps/">Maps</a></li>
-    <li><a href="http://web.iastate.edu/safety/">Safety</a></li>
+    <li><a href="https://info.iastate.edu/">Directory</a></li>
+    <li><a href="https://www.fpm.iastate.edu/maps/">Maps</a></li>
+    <li><a href="https://web.iastate.edu/safety/">Safety</a></li>
     <li class="isu-navbar_dropdown_signons isu-navbar_dropdown" aria-expanded="false">
-      <a href="https://www.it.iastate.edu/signons/" class="isu-navbar_dropdown-toggle" role="button" aria-haspopup="true" aria-expanded="false">Sign Ons</a>
+      <a href="https://web.iastate.edu/signons" class="isu-navbar_dropdown-toggle" role="button" aria-haspopup="true" aria-expanded="false">Sign Ons</a>
       <ul class="isu-navbar_dropdown-menu isu-navbar_signons">
-        <li><a href="http://accessplus.iastate.edu/">AccessPlus</a></li>
-        <li><a href="http://bb.its.iastate.edu/">Blackboard</a></li>
-        <li><a href="http://iastate.box.com/">CyBox</a></li>
+        <li><a href="https://accessplus.iastate.edu/">AccessPlus</a></li>
+        <li><a href="https://canvas.iastate.edu/">Canvas</a></li>
+        <li><a href="https://iastate.box.com/">CyBox</a></li>
         <li><a href="http://cymail.iastate.edu/">CyMail</a></li>
```

Comment:
Sad that this URL redirect doesn't support https. Nothing we can do about it, but it caught my eye because it was the only non-https url.